action.yaml

```diff
@@ -18,10 +17,9 @@ inputs:
   changed-files-path:
     description: The path to the file containing the changed files
     required: false
-
 runs:
   using: docker
-  image: Dockerfile
+  image: docker://public.ecr.aws/vdb-public/gh-mpyl:latest
```

Comment:
This will be replaced by an actual tag every time a new change is merged.

Reply:
Is it also used as is? If not, would something like `{placeholder}` be more clear?

Reply:
The value of `runs.image` determines which version of the `gh-mpyl` docker image used. Every time a release happens, the workflow modifies this line by [replacing it with the released version](https://github.com/Vandebron/gh-mpyl/pull/29/files#diff-eecbed2b66fe9be749f388e527e472f10b086c8cced6e90c234c8a23be5b8798R81-R84).

This particular change is not necessary at all, I could've left this file exactly as it currently is, or even replace it with
```yaml
runs:
  image: foo
```
and this would've had no impact at all in this PR.

I only changed it to bring attention to it 🙂